sound/pulseaudio/patches/020-doxygen.patch

```diff
@@ -1,10 +1,10 @@
 --- a/meson.build
 +++ b/meson.build
-@@ -698,7 +698,6 @@ check_dep = dependency('check', version
- 
+@@ -827,7 +827,6 @@ check_dep = dependency('check', version
  # Subdirs
  
--subdir('doxygen')
- subdir('po')
- if get_option('man')
-   subdir('man')
+ if get_option('doxygen')
+-  subdir('doxygen')
```

Comment:
this is definitely not needed anymore.

Reply:
This patch was ready for it to build somehow.

Reply:
Pass -Ddoxygen=false